### ISSUE TITLE
[HU1] Visualización de características de una serie

### ISSUE THREAD

**Issue by Parka015** [Parka015/IV-Proyecto#2]
Como usuario me gustaría poder ver si una serie tendrá terror, romance, acción, suspense, comedia o tiene partes musicales. Y que además, pueda saber por medio de alguna escala cuanta "cantidad" de cada género contiene

**Comment by Parka015** [comment]
Simplemente intento imaginar lo que me diría un usuario si le entrevisto preguntándole que información quiere que se guarde en la estructura que almacene a la serie, porque lo veo como un dato importante de cara a que el programador sepa por donde empezar, el objetivo que persigue este HU es como páginas al estilo de FilmAffinity, que sirve para que uno sepa si le va a interesar una serie o pelicula, en mi caso, solo trato series. Asi que supongo que manteniendo lo que tengo y aclarando con que finalidad el usuario quiere este HU debería estar bien

**Comment by JJ** [comment]
Claro, siempre los detalles se piensan entre los dos, faltaría más.
